src/tools/r2rdump/Amd64/GcTransition.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml.Serialization;
+
+namespace R2RDump.Amd64
+{
+    public struct InterruptibleRange
+    {
+        [XmlAttribute("Index")]
+        public uint Index { get; set; }
+        public uint StartOffset { get; set; }
+        public uint StopOffset { get; set; }
+        public InterruptibleRange(uint index, uint start, uint stop)
+        {
+            Index = index;
+            StartOffset = start;
+            StopOffset = stop;
+        }
+    }
+
+    public class GcTransition : BaseGcTransition
+    {
+        public int SlotId { get; set; }
+        public bool IsLive { get; set; }
+        public int ChunkId { get; set; }
+
+        public GcTransition() { }
+        public string SlotState { get; set; }
```

Comment:
nit: Can you move the SlotState declaration up where the rest of the Properties are declared?